packages/iov-lisk/src/derivation.spec.ts

```diff
@@ -32,4 +32,28 @@ describe("Derivation", () => {
       expect(pubkeyToAddress(pubkey)).toEqual("10176009299933723198L");
     });
   });
+
+  describe("isValidAddress", () => {
+    it("works for valid numbers within unsigned int64 range", () => {
+      expect(isValidAddress("1234567890L")).toBeTruthy();
+      expect(isValidAddress("6076671634347365051L")).toBeTruthy();
+      expect(isValidAddress("10176009299933723198L")).toBeTruthy();
+      // 2**64-1
+      expect(isValidAddress("18446744073709551615L")).toBeTruthy();
+    });
+
+    it("rejects malformed addresses", () => {
+      // invalid ending
+      expect(isValidAddress("1234567890R")).toBeFalsy();
```

Comment:
`toBeFalsy()` -> `toEqual(false)` because we don't want 0, null, "" or undefined to pass